ci/release.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash -e
 # Releases this Puppet module to the Puppet Forge
 
-docker build -t puppet-pdk -f Dockerfile.pdk .
+# To ensure this script always executes relative to the repo root
+cd "$(dirname "$0")/.."
+
+docker build -t puppet-pdk -f ./ci/Dockerfile.pdk .
 
 summon docker run --rm -t \
   -v $PWD:/conjur \
```

Comment:
does this need to point to the secrets.yml file that's now in `ci/secrets.yml`?

Reply:
Thank you! Good catch.

Reply:
@izgeri Sorted.